pkg/mcp/snapshot/snapshot.go

```diff
@@ -43,6 +46,12 @@ type Info struct {
 	Names []string
 }
 
+// Object is used for configz to show the mcp resource
+type Object struct {
```

Comment:
Can you use
https://github.com/istio/istio/blob/1e88585345b259d23f67d957befeecd08ac516c4/pkg/mcp/sink/sink.go#L248

Reply:
Done.